src/egon/data/datasets/power_plants/wind_offshore.py

```diff
@@ -8,6 +8,140 @@
+        'SEL960138676303': 'Büttel',
+        'SEL964485571465': 'Cloppenburg',
+        'SEL964827249373': 'Emden/Ost',
+        'SEL969972182469': 'Büttel',
+        'SEL988334563685': 'Dörpen/West',
+        'SEL990492206410': 'Bentwisch',
+        'SEL997728577467': 'Lubmin',
+        'SEL998582120202': 'Emden/Ost',
+        }
+    return to_city
+
+def map_id_bus(scenario="eGon2035"):
+    # Import manually generated list of wind offshore farms with their
+    # connection points (OSM_id)
+    if scenario=="eGon2035":
+        id_bus = {
+            "Büttel": "136034396",
+            "Heide/West": "603661085",
+            "Suchraum Gemeinden Ibbenbüren/Mettingen/Westerkappeln": "114319248",
+            "Suchraum Zensenbusch": "76185022",
+            "Rommerskirchen": "24839976",
+            "Oberzier": "26593929",
+            "Garrel/Ost": "23837631",
+            "Diele": "177829920",
+            "Dörpen/West": "142487746",
+            "Emden/Borßum": "34835258",
+            "Emden/Ost": "34835258",
+            "Hagermarsch": "79316833",
+            "Hanekenfähr": "61918154",
+            "Inhausen": "29420322",
+            "Unterweser": "32076853",
+            "Wehrendorf": "33411203",
+            "Wilhelmshaven 2": "23744346",
+            "Rastede": "23837631",
+            "Bentwisch": "32063539",
+            "Lubmin": "460134233",
+            "Suchraum Gemeinde Papendorf": "32063539",
+            "Suchraum Gemeinden Brünzow/Kemnitz": "460134233",
+            "inhausen": "29420322",
+            "Cloppenburg": "50643382"
+        }
+    elif scenario=="status2019":
+        id_bus = {
+        'UW Inhausen':'29420322',
+        'UW Bentwisch':'32063539',
+        'UW Emden / Borssum':'34835258',
+        'UW Emden Ost':'34835258',
+        'UW Cloppenburg':'50643382',
+        'UW Hagermarsch':'79316833',
+        'UW Büttel':'136034396',
+        'UW Dörpen West':'142487746',
+        'UW Diele':'177829920',
+        'UW Lubmin':'460134233',
+        }
+    else:
+        id_bus = {}
+        
+    return id_bus
+
+def map_w_id():
+    w_id = {
+        "Büttel": "16331",
+        "Heide/West": "16516",
+        "Suchraum Gemeinden Ibbenbüren/Mettingen/Westerkappeln": "16326",
+        "Suchraum Zensenbusch": "16139",
+        "Rommerskirchen": "16139",
+        "Oberzier": "16139",
+        "Garrel/Ost": "16139",
+        "Diele": "16138",
+        "Dörpen/West": "15952",
+        "Emden/Borßum": "15762",
+        "Emden/Ost": "16140",
+        "Hagermarsch": "15951",
+        "Hanekenfähr": "16139",
+        "Inhausen": "15769",
+        "Unterweser": "16517",
+        "Wehrendorf": "16139",
+        "Wilhelmshaven 2": "16517",
+        "Rastede": "16139",
+        "Bentwisch": "16734",
+        "Lubmin": "16548",
+        "Suchraum Gemeinde Papendorf": "16352",
+        "Suchraum Gemeinden Brünzow/Kemnitz": "16548",
+        "inhausen": "15769",
+        "Cloppenburg": "16334"
+    }
+    return w_id
```

Comment:
The weather_ids are added differently, this function is not used at all. Can we remove it?